tracing/tracing.go

```diff
@@ -142,6 +177,7 @@ func New(service string, host string, zipkinEndpoint string) io.Closer {
 		reporter,
 		zipkin.WithLocalEndpoint(localEndpoint),
 		zipkin.WithSharedSpans(false),
+		zipkin.WithSampler(zipkin.AlwaysSample),
```

Comment:
Is this correct? do we alaways want to sample or are you relying on some kind of override?

Reply:
ah, no, thanks, I need to remove that ...

Reply:
removed